libs/CCAppCommon/devices/3dConnexion/cc3DMouseManager.cpp

```diff
@@ -90,9 +90,11 @@ void cc3DMouseManager::enableDevice(bool state, bool silent)
 
 void cc3DMouseManager::releaseDevice()
 {
+    // nothing to release
```

Comment:
general comment: for historical reasons, we use tabulations (4 spaces) and not spaces in this project. This is why all your comments appear shifted in github.